kubernetes/loculus/values.yaml

```diff
@@ -101,7 +91,24 @@ defaultOrganismConfig: &defaultOrganismConfig
             releaseDate: ncbiReleaseDate
           args:
             fieldType: dateRangeUpper
+        rangeOverlapSearch:
+          rangeName: sampleCollectionDateRange
+          rangeDisplayName: Collection date
+          bound: upper
         noInput: true
+      - name: sampleCollectionDate
+        displayName: Collection date (text)
```

Comment:
Is the displayName also used as the column title?

Reply:
So we can add `notSearchable: true` for search.

In terms of columns it looks like we have a bug here in that there is a `Collection date` in the column selector which when ticked doesn't make a column appear. (For understandable reasons!)

Reply:
> In terms of columns it looks like we have a bug here in that there is a Collection date in the column selector which when ticked doesn't make a column appear

Should we just tolerate this and make an issue for it and try and adress soonish? Besides this I think the solution is good.

Reply:
I removed the "(text)" and added the "notSearchable" bit

Reply:
(i.e. we can remove it specifically by name/id or whatever in a totally hardcoded way)

Reply:
hacky fix: https://github.com/loculus-project/loculus/pull/3308/commits/4cc6f4a59f225b7aaf55e3a406900d43b543805d

issue: https://github.com/loculus-project/loculus/issues/3325

`notSearchable` didn't actually work, I fixed it here: https://github.com/loculus-project/loculus/pull/3308/commits/2cca86efecc3cba09fb0e0dffa64ea00ca1ee00c - not sure if this might have other consequences we didn't yet think about?